src/Workspaces/Remote/ServiceHub/Shared/RemoteEndPoint.cs

```diff
@@ -216,11 +217,16 @@ public async Task<T> InvokeAsync<T>(string targetName, IReadOnlyList<object?> ar
             }
         }
 
-        public static Task WriteDataToNamedPipeAsync<TData>(string pipeName, TData data, Func<ObjectWriter, TData, CancellationToken, Task> dataWriter, CancellationToken cancellationToken)
+#pragma warning disable CA1200 // Avoid using cref tags with a prefix (the compiler cannot otherwise resolve this reference)
```

Comment:
>#pragma warning disable CA1200 // Avoid using cref tags with a prefix (the compiler cannot otherwise resolve this reference [](start = 0, length = 123)

Why is CA1200 reported here - ObjectWriter type is referenced in the signature.

Reply:
There is more than one `Roslyn.Utilities.ObjectWriter` type. Only one of them is accessible to code in this file, but `cref` attributes ignore accessibility.